chart/inception/README.md

```diff
@@ -0,0 +1,177 @@
+| auth.oauth2.clientSecret.value | string | `""` | OAuth2 client secret in plain text (**NOT RECOMMENDED FOR PRODUCTION**).    Ignored if a reference to an external secret is specified. |
+| auth.oauth2.clientSecret.existingSecret.secretName | string | `""` | Name of the secret holding the OAuth2 client secret value. |
+| auth.oauth2.clientSecret.existingSecret.secretKey | string | `""` | Key inside the secret holding the OAuth2 secret's value. |
+| auth.oauth2.scope | string | `""` | OAuth2 scope. |
+| auth.oauth2.authorizationGrantType | string | `""` | OAuth2 grant type. |
+| auth.oauth2.redirectURI | string | `""` | OAuth2 redirect URI. |
+| auth.oauth2.issuerURI | string | `""` | OAuth2 Issuer URI. |
+| auth.oauth2.usernameAttribute | string | `""` | OAuth2 claim holding the username attribute. |
+| auth.saml2.enabled | bool | `false` | Enables SAML2 SSO login. |
+| auth.saml2.autoLogin | bool | `false` | Enables SAML2 as default login method (skips INCEpTION's default login page). |
+| auth.saml2.signRequest.enabled | bool | `false` | Enables SAML2 request signing. |
+| auth.saml2.signRequest.certificateValue | string | `""` | Public certificate value un plain text.    Ignored if a reference to an external TLS Secret is specified. |
+| auth.saml2.signRequest.privateKeyValue | string | `""` | Private Key value, in plain text (**NOT RECOMMENDED FOR PRODUCTION**).    Ignored if a reference to an external secret is specified.  |
+| auth.saml2.signRequest.existingTlsSecret | string | `""` | Reference to an external Kubernetes TLS secret in the same namespace holding the certificate/private key pair to sign SAML2 requests. |
+| auth.saml2.assertingPartyCertificate.value | string | `""` | Certificate in plain text.     Ignored if a reference to an external Secret is specified. |
+| auth.saml2.assertingPartyCertificate.existingSecret.secretName | string | `""` | Name of the secret holding the Identity Provider certificate. |
+| auth.saml2.assertingPartyCertificate.existingSecret.secretKey | string | `""` | Key inside the secret holding the Identity Provider certificate. |
+| auth.saml2.assertingPartyEntityID | string | `""` | Identity Provider's SAML2 Asserting ID. |
+| auth.saml2.assertingPartySSOUrl | string | `""` | Identity Provider's SAML2 SSO URL. |
+| auth.preAuthentication.enabled | bool | `false` | Enables the PreAuthentication feature. |
+| auth.preAuthentication.headerPrincipal | string | `""` | Name of the header holding the remote user name. |
+| auth.preAuthentication.newUserRole | string | `""` | Role given to new pre-authenticated users. |
+| auth.preAuthentication.customUserRoles | object | `{}` | Roles to map to dedicated pre-authenticated users, in the `username: role` format. |
+
+### INCEpTION deployment settings
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| replicaCount | int | `1` | Number of replicas fir INCEpTION deployment. |
```

Comment:
Should something be said here that multiple replica using the same DB / filesystem location are not possible or does the replica count include replicas of the DB / FS as well?